velum.yaml

```diff
@@ -75,3 +75,10 @@ spec:
     env:
     - name: MYSQL_ROOT_PASSWORD # FIXME: do not set root password, create user/password
       value: "salt"             # FIXME
+    volumeMounts:
+      - mountPath: /var/lib/mysql
+        name: velum-mariadb-data
+  volumes:
+    - name: velum-mariadb-data
+      hostPath:
```

Comment:
I thought that these manifests were meant for production-ready deployments. If so refrain from using `hostPath`. That being said, maybe you could just add a comment here and add a note in the README or something ?